changelog/items/key-updates/json_payloads.md

```diff
@@ -0,0 +1 @@
+- BREAKING: All non-GET handlers now read their parameters from the request's body JSON instead of the form.  
```

Comment:
Calling out as a reminder we should bump the major version then

Reply:
Can do. But we can also not do it - the major version is zero, which implies no compatibility guarantees. 

But it might be time this service got its recognition with an official version. :)

Reply:
I think it makes sense for the official version as it will align with us pushing community portals

Reply:
Should this be a follow up for when we're ready to release?

Reply:
Version 1.0!